docs/src/modules/utils/helpers.js

```diff
@@ -57,9 +57,10 @@ const packagesWithBundledTypes = ['@material-ui/core', '@material-ui/lab'];
  * @param {Record<string, string>} deps - list of dependency as `name => version`
  */
 function addTypeDeps(deps) {
-  const packagesWithDTPackage = Object.keys(deps).filter(
-    (name) => packagesWithBundledTypes.indexOf(name) === -1,
-  );
+  const packagesWithDTPackage = Object.keys(deps)
+    .filter((name) => packagesWithBundledTypes.indexOf(name) === -1)
+    // All the Material-UI packages come with bundled types
```

Comment:
No they don't. The existing logic is sound. Why not just add the new package to the existing logic?

Reply:
> No they don't.

Do you have an example?

> Why not just add the new package to the existing logic?

Assuming all the Material-UI packages **should** come with bundled types, why spend mental energy on maintaining such list?

Note that codesandbox is silent when a wrong package is included. For instance, take this demo https://material-ui.com/system/basics/#demo, open it to codesandbox, and look at the installed dependencies.

Reply:
`@material-ui/utils`.

Reply:
>  For instance, take this demo material-ui.com/system/basics/#demo, open it to codesandbox, and look at the installed dependencies.

Yes there's a bug and infra in place to fix it.

Reply:
> @material-ui/utils

Cool, I will fix that so we can merge this pull request as is.

> @material-ui/utils

<img width="410" alt="Capture d’écran 2020-08-26 à 11 26 42" src="https://user-images.githubusercontent.com/3165635/91286693-0c3ec000-e78f-11ea-9da1-9964fd5e1a71.png">

Reply:
The assumption is still wrong. I don't understand why we can't keep the correct assumption and extend the packages that do ship types? Adding types to /utils is not really worth it at this this time.

Reply:
The current approach forces to keep track of a list of packages that support TypeScript. Assuming we will add 10 new packages, I don't see the value of having to think about it when we can go all-in on TypeScript. The value of TypeScript support of /utils is so material-ui-x can use the package more efficiently. e.g. https://github.com/mui-org/material-ui-x/blob/fc207098f9c919faa88ec5cb827a37f5d7b87e9b/packages/grid/x-grid/src/XGrid.tsx#L3

Reply:
So in both cases, it doesn't work, we might as well well pick the approach that has a chance to eventually work.